### PR TITLE
Fix displaying subject identifier checkboxes

### PR DESCRIPTION
**Pull request by yathindrakodithuwakku**
### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- Related PR `#1` or (None)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authentication-step.tsx [hunk 1]
```diff
@@ -137,6 +137,14 @@ export const AuthenticationStep: FunctionComponent<AuthenticationStepPropsInterf
 
     const classes = classNames("authentication-step-container timeline-body", className);
 
+    let isContainsOTPAuthenticator = false;
+
+    step.options.map(option => [
+        IdentityProviderManagementConstants.TOTP_AUTHENTICATOR, 
+        IdentityProviderManagementConstants.EMAIL_OTP_AUTHENTICATOR, 
+        IdentityProviderManagementConstants.SMS_OTP_AUTHENTICATOR ]
+        .includes(option.authenticator) && (isContainsOTPAuthenticator = true));
```
Comment (thivi) [inline apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authentication-step.tsx:146]: I think it is better to move this `(isContainsOTPAuthenticator = true)` into an `if` block.

Comment (yathindrakodithuwakku) [reply apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authentication-step.tsx]: Done

**Review by thivi — APPROVED** [review]

File: apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authentication-step.tsx [hunk 2]
```diff
@@ -137,6 +137,18 @@ export const AuthenticationStep: FunctionComponent<AuthenticationStepPropsInterf
 
     const classes = classNames("authentication-step-container timeline-body", className);
 
+    let isContainsOTPAuthenticator = false;
```
Comment (brionmario) [inline apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authentication-step.tsx:140]: better to explicitly define the type.
```
let isContainsOTPAuthenticator: boolean = false;
```

File: apps/console/src/features/identity-providers/constants/identity-provider-management-constants.ts [hunk 3]
```diff
@@ -273,6 +273,7 @@ export class IdentityProviderManagementConstants {
     public static readonly FIDO_AUTHENTICATOR: string = "FIDOAuthenticator";
     public static readonly BASIC_AUTHENTICATOR = "BasicAuthenticator";
     public static readonly IDENTIFIER_FIRST_AUTHENTICATOR = "IdentifierExecutor";
+    public static readonly SMS_OTP_AUTHENTICATOR = "sms-otp";
```
Comment (brionmario) [inline apps/console/src/features/identity-providers/constants/identity-provider-management-constants.ts:276]: Please add types to the above missing ones too.
```
public static readonly SMS_OTP_AUTHENTICATOR: string = "sms-otp";
```

**Pull request merged**